tracks/python/exercises/raindrops/mentoring.md

```diff
@@ -0,0 +1,127 @@
+### Problem and challenges
```

Comment:
```suggestion
### Problem and Challenges
```

Reply:
(Either would work for me, as long as it's consistent in the track)